### ISSUE TITLE
Issue found on page 'DuckDB with DataFusion'

### ISSUE THREAD

**Issue by szarnyasg** [duckdb/duckdb-web#1528]
_Please describe the issue on the page and include the "Page URL" link shown below._

Page URL: <https://duckdb.org/docs/guides/python/datafusion.html>

The example on the page doesn't work. Run it and you will get:

```text
Traceback (most recent call last):
    ...
    .aggregate(
     ^^^^^^^^^^
Exception: Schema error: No field named a. Valid fields are c85c1cb92bf6c465ca501674bc454f7f1."A", c85c1cb92bf6c465ca501674bc454f7f1.fruits, c85c1cb92bf6c465ca501674bc454f7f1."B", c85c1cb92bf6c465ca501674bc454f7f1.cars.

```

```python
import duckdb
import pyarrow as pa
import pandas as pd
import datafusion as df
from datafusion import functions as f

pandas_df = pd.DataFrame(
    {
        "A": [1, 2, 3, 4, 5],
        "fruits": ["banana", "banana", "apple", "apple", "banana"],
        "B": [5, 4, 3, 2, 1],
        "cars": ["beetle", "audi", "beetle", "beetle", "beetle"],
    }
)

arrow_table = table = pa.Table.from_pandas(pandas_df)
arrow_batches = table.to_batches()

ctx = df.SessionContext()
datafusion_df = ctx.create_dataframe([arrow_batches])
print(datafusion_df)

arrow_batches = (
    datafusion_df
        .aggregate(
            [df.col("fruits")],
            [f.sum(df.col("A")).alias("sum_A_by_fruits")]
        )
        .sort(df.col("fruits").sort(ascending=True))
        .collect()
)
datafusion_to_arrow = (
    pa.Table.from_batches(arrow_batches)
)
print(datafusion_to_arrow)
```

**Comment by Mytherin** [comment]
I think we can just remove this entire page perhaps?

**Comment by szarnyasg** [comment]
Resolved by #1578.
